src/components/common/PopoverMenu.tsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import { ListItem, MenuItem, MenuList, Popover, useTheme } from '@mui/material';
+import { DropdownItem } from '@terraware/web-components/components/Dropdown';
+
+type PopoverMenuProps = {
+  items: DropdownItem[] | [];
+  handleClick: (item: DropdownItem) => void;
+  otherItems?: DropdownItem[];
```

Comment:
can we make items and otherItems just sections, each section is a list of menu itmes and a handeclick.
sections are separated by '---'
this way it's not very specifically coded for the org menu create.
example:
```
type MenuSection = {
  handleClick: (item: DropdownItem) => void;
  items: DropdownItem[];
}
```
PopoverMenuProps can then be:
```
  menuSections: MenuSection[];
  anchorElement: ....
```

Reply:
The other option is to have a single list of menuitems and support a 'divider' item in the list

Reply:
Actually - a single list of menuitems, and if item is null - we could assume a divider (with some comments in the file explaining this behavior).